lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -1722,6 +1742,8 @@ ThreadSP ProcessGDBRemote::SetThreadStopInfo(
     } else {
       bool handled = false;
       bool did_exec = false;
+      if (reason == "none")
```

Comment:
This definitely needs a comment, or maybe somewhere else saying that none == empty, so we always convert none to empty.

Reply:
I updated the patch where I handle this a little more cleanly, checking that `reason` has a value, and that the value is not `"none"`, and added a short comment.